main.go

```diff
@@ -18,23 +18,31 @@ import (
 )
 
 var (
-	name       string // plugin name
-	repo       string // plugin repository
-	ref        string // plugin repository reference
-	sha        string // plugin repository commit
-	kind       string // plugin kind (action, bitrise, harness)
-	outputFile string // plugin outputfile
+	name   string // plugin name
+	repo   string // plugin repository
+	ref    string // plugin repository reference
+	sha    string // plugin repository commit
+	kind   string // plugin kind (action, bitrise, harness)
+	dryRun bool   // plugin won't be executed on setting this flag. Only source will be downloaded. Used for caching the plugin dependencies
```

Comment:
This is a misnomer. This should probably be called `downloadOnly` or similar.

Reply:
Done